packages/php/src/HAR/Payload.php

```diff
@@ -94,7 +94,7 @@ private function processRequest(Request $request): array
                     static::convertFileObjectForArray($this->sanitizeInputPerConfig($request->allFiles()))
                 ),
             ];
-        } elseif ($request->getContentType() === 'form') {
+        } elseif (in_array($content_type, ['application/x-www-form-urlencoded', 'multipart/form-data'])) {
```

Comment:
Switched to doing it this way because `getContentType()` is deprecated in 6.2, and `getContentTypeFormat()` doesn't exist in 6.0.

Reply:
I really cannot comprehend why they removed `getContentType()`.

Reply:
I think they said it's confusing because it's called getContentType which you'd expect to return the `content-type` header but it actually returns with some internally formatted hash, like `json` instead of `application/json`

Reply:
For some reason this broke the integration tests... https://github.com/readmeio/metrics-sdks/actions/runs/4609961954/jobs/8147819159

So i'm just gunna ignore the deprecation for now and bring the code back to what it was before: 51d1780b13439da600808766d9e1b36054094b86 :yolo: